Google.PowerShell/Compute/GceNetworkCmdlets.cs

```diff
@@ -215,4 +217,74 @@ private IEnumerable<Network> GetAllProjectNetworks()
             } while (!Stopping && request.PageToken != null);
         }
     }
+
+    /// <summary>
+    /// <para type="synopsis">
+    /// Removes Google Compute Engine networks.
+    /// </para>
+    /// <para type="description">
+    /// Removes one or more Google Compute Engine networks. Will raise errors if the networks do not exist.
+    /// The cmdlet will delete the networks in the default project if -Project is not used.
+    /// </para>
+    /// <example>
+    ///   <code>PS C:\> Remove-GceNetwork -Network "my-network"</code>
+    ///   <para>This command removes network "my-network" in the default project.</para>
+    /// </example>
+    /// <example>
+    ///   <code>PS C:\> Remove-GceNetwork -Network "my-network1", "my-network2" -Project "my-project"</code>
+    ///   <para>
+    ///   This command removes 2 networks ("my-network1" and "my-network2") in the project "my-project".
+    ///   </para>
+    /// </example>
+    /// <para type="link" uri="(https://cloud.google.com/compute/docs/vpc/)">
+    /// [Networks]
+    /// </para>
+    /// </summary>
+    [Cmdlet(VerbsCommon.Remove, "GceNetwork", SupportsShouldProcess = true)]
+    public class RemoveGceNetwork : GceCmdlet
+    {
+        /// <summary>
+        /// <para type="description">
+        /// The project to remove the networks in. If not set via PowerShell parameter processing, will
+        /// default to the Cloud SDK's DefaultProject property.
+        /// </para>
+        /// </summary>
+        [Parameter(Mandatory = false)]
+        [ConfigPropertyName(CloudSdkSettings.CommonProperties.Project)]
+        public override string Project { get; set; }
+
+        /// <summary>
+        /// <para type="description">
+        /// The names of the networks to be removed.
+        /// </para>
+        /// </summary>
+        [Parameter(Mandatory = true, Position = 0, ValueFromPipelineByPropertyName = true)]
+        [ArrayPropertyTransform(typeof(Network), nameof(Apis.Compute.v1.Data.Network.Name))]
```

Comment:
I think this is another case where we need to have a separate parameter set for taking the object. The object contains the project it is a part of in its selfLink.